### PR TITLE
Add missing Python examples

### LINKED ISSUE #1587: [Task]: Add Shimmer Python Code Examples
## Description

Shimmer code examples using the Python binding to showcase the library's functionalities in all languages. 

## Motivation

As we aim to make developer onboarding easy in all languages, it'd be great to provide the same examples in all languages.

## Requirements

Add examples showcasing the following functionalities using Rust:

- [x] Get funds
- [x] Send microtransactions
- [x] List transactions
- [x] Claim outputs
- [x] Mint Native tokens
- [x] Melt Native tokens
- [x] Burn native tokens
- [x] Mint NFTs
- [x] Send NFTs
- [x] Burn NFTs

## Are you planning to do it yourself in a pull request?

If no one's planning on working on these, I can take a go at it.

### PR DESCRIPTION
**Pull request by samuel-rufi**
Fixes https://github.com/iotaledger/wallet.rs/issues/1587

### CODE REVIEW

File: bindings/python/examples/burn_native_tokens.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from iota_wallet import IotaWallet
+import time
```
Comment (Thoralf-M) [inline bindings/python/examples/burn_native_tokens.py]: You import this in all examples, but it's only used in the last one?

File: bindings/python/examples/burn_nft.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from iota_wallet import IotaWallet
+import time
+
+# In this example we will burn native tokens
```
Comment (Thoralf-M) [inline bindings/python/examples/burn_nft.py]:
```suggestion
# In this example we will burn an NFT
```

File: bindings/python/examples/destroy_foundry.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from iota_wallet import IotaWallet
+import time
+
+# In this example we will burn native tokens
```
Comment (Thoralf-M) [inline bindings/python/examples/destroy_foundry.py]:
```suggestion
# In this example we will destroy a foundry
```

File: bindings/python/examples/burn_native_tokens.py (hunk 1 above)
Comment (samuel-rufi) [reply bindings/python/examples/burn_native_tokens.py]: good eyes

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**